cxx-squid/src/main/java/org/sonar/cxx/visitors/CxxFileLinesVisitor.java

```diff
@@ -71,6 +89,9 @@ public void visitToken(Token token) {
       String[] tokenLines = token.getValue().split("\n", -1);
       for (int line = token.getLine(); line < token.getLine() + tokenLines.length; line++) {
         linesOfCode.add(line);
+//        if (!token.isGeneratedCode()) {
```

Comment:
Maybe not but I did not see any coverage result until now. Do you know how functional macros are handled by the coverage tools?